src/NServiceBus.MessagingBridge/Configuration/BridgeConfiguration.cs

```diff
@@ -110,14 +115,31 @@ internal FinalizedBridgeConfiguration FinalizeConfiguration(ILogger<BridgeConfig
             {
                 var sb = new StringBuilder();
 
-                sb.AppendLine("Events can only be associated with a single publisher, please verify subscriptions for:");
-                sb.AppendLine();
+                if (allowMultiplePublishersSameEvent)
+                {
+                    sb.Append(
+                        "The following subscriptions with multiple registered publishers are ignored as best practices are not enforced:\r");
+                }
+                else
+                {
+                    sb.Append(
+                        "Events can only be associated with a single publisher, please verify subscriptions for:\r");
+                }
+
                 foreach (var eventType in eventsWithMultiplePublishers)
                 {
                     var publishers = string.Join(", ", eventType.Select(e => e.Publisher));
-                    sb.AppendLine($"- {eventType.Key}, registered publishers: {publishers}");
+                    sb.Append($"- {eventType.Key}, registered publishers: {publishers}\r");
+                }
+
+                if (allowMultiplePublishersSameEvent)
+                {
+                    logger.LogWarning(sb.ToString());
```

Comment:
This should also be adjusted to structured logging. Splitting the log event and its payload.